thoth/storages/graph/postgres.py

```diff
@@ -4495,7 +4495,7 @@ def sync_package_analysis_result(self, document: dict) -> None:
             self._session.commit()
 
     def _get_or_create_python_package_index(
-        self, index_url: str, only_if_enabled: bool = True
+        self, index_url: str, only_if_enabled: bool = False
```

Comment:
I meant replacing line:

```python
    index = self._session.query(PythonPackageIndex).filter_by(url=index_url).one()
```

with:

```python
    index = self._get_or_create_python_package_index(index_url, only_if_enabled=False)
```

in `create_python_cve_record` method.